src/openrct2-ui/windows/EditorScenarioOptions.cpp

```diff
@@ -742,55 +742,51 @@ static void window_editor_scenario_options_financial_invalidate(rct_window* w)
  */
 static void window_editor_scenario_options_financial_paint(rct_window* w, rct_drawpixelinfo* dpi)
 {
-    int32_t x, y;
+    ScreenCoordsXY screenCoords;
```

Comment:
```suggestion
    ScreenCoordsXY screenCoords{};
```
Initialize it, just to be sure